src/config/config.rs

```diff
@@ -217,6 +218,7 @@ pub struct Field {
   pub doc: Option<String>,
   pub modify: Option<ModifyField>,
   pub inline: Option<InlineType>,
+  pub added_field: Option<AddedFieldType>,
```

Comment:
`@field` should only be available on types and not on the field itself. This information should be inferred inside of from_config and not saved in the original config.